rdt/transformers/numerical.py

```diff
@@ -241,7 +244,7 @@ def fit(self, data):
 
     def _copula_transform(self, data):
         cdf = self._univariate.cdf(data)
-        return scipy.stats.norm.ppf(cdf.clip(0 + EPSILON, 1 - EPSILON))
+        return scipy.stats.norm.ppf(cdf.clip(0 + 1e-5, 1 - 1e-5))
```

Comment:
Let's define EPSILON like it's done in Copulas: https://github.com/sdv-dev/Copulas/blob/3210fc141c741185b781686cb69e00a96972d960/copulas/__init__.py#L16

```python
EPSILON = np.finfo(np.float32).eps
```

We can put this at module level.